core/src/org/pentaho/di/core/Const.java

```diff
@@ -1357,6 +1357,30 @@ public static String rightPad( StringBuffer ret, int limit ) {
   }
 
   /**
+   * Right pad a StringBuilder: adds spaces to a string until a certain length. If the length is smaller then the limit
+   * specified, the String is truncated.
+   *
+   * @param ret
+   *          The StringBuilder to pad
+   * @param limit
+   *          The desired length of the padded string.
+   * @return The padded String.
+   */
+  public static String rightPad( StringBuilder ret, int limit ) {
```

Comment:
I checked into that. CharSequence doesn't have mutators. So, we'd not be able to mutate it unless we did "instanceof" checks and casts. I also checked their superclass (AbstractStringBuilder), but that's package-protected so I couldn't work with that either. I opted for copy/paste which I thought was better than "instanceof" stuff IMO.